### PR TITLE
text-widget

### PR DESCRIPTION
**Pull request by kghamilton89**

### CODE REVIEW

File: widgets/text-widget/README.md [hunk 1]
```diff
@@ -1,61 +1,72 @@
...
-|Icon Position|`icon-position = bottom`|Position of the icon relative to the series value.<br>Possible settings: `left`, `right`, `bottom`, `top`|[![](./images/button.png)](https://apps.axibase.com/chartlab/f95d6995/2/)|
-|Icon Color|`icon-color = orange`|Color of the icon.|[![](./images/button.png)](https://apps.axibase.com/chartlab/f95d6995/3/)|
-|Icon Alert Style|`icon-alert-style = fill: red;`|Alert styles applied to the icon when the `alert-expression` is satisfied.|[![](./images/button.png)](https://apps.axibase.com/chartlab/7e91b333/21/)|
+* The settings apply to the `[widget]` section.
+* [Shared](../shared/README.md#widget-settings) `[widget]` syntax is inherited.
 
-## Examples
+Name|Example|Description|&nbsp;
+:--|:--|:--|:--
+<a name="circle"></a>[`circle`](#circle)|`circle = true`|Display series values inside a circle.<br>Boolean expression.<br>Default value: `false`|[↗](https://apps.axibase.com/chartlab/0f9d1664)
```
Comment (rodionos) [inline widgets/text-widget/README.md]: Please see earlier notes regarding where it's ok to say Boolean expression.

File: widgets/text-widget/README.md [hunk 2]
```diff
@@ -1,61 +1,72 @@
...
+<a name="min-font-size"></a>[`min-font-size`](#min-font-size)|`min-font-size = 20`|Minimum font size for labels, in pixels.|[↗](https://apps.axibase.com/chartlab/99035024)
+<a name="max-font-size"></a>[`max-font-size`](#max-font-size)|`max-font-size = 60`|Maximum font size for labels, in pixels.|[↗](https://apps.axibase.com/chartlab/00379a76)
+<a name="on-series-click"></a>[`on-series-click`](#on-series-click)|`on-series-click = callDialog({ type: 'page', url: 'https://axibase.com/docs/atsd/' })`|Specify behavior upon series value click.<br>See example configuration for possible values.<br>Default value: `time chart`|[↗](https://apps.axibase.com/chartlab/db0b293e)
```
Comment (rodionos) [inline widgets/text-widget/README.md]: There is no text default value but by default a dialog with time chart is open. `time chart` or `anyword` is just ignored. If the value has no impact  - it's not a default value.

File: widgets/text-widget/README.md [hunk 3]
```diff
@@ -1,61 +1,72 @@
...
 
-### Multiple Series
+### Series Settings
 
-![](./images/text-widget-3.png)
+* The settings apply to the `[series]` section.
+* [Shared](../shared/README.md#series-settings) `[series]` syntax is inherited.
 
-[![](./images/button.png)](https://apps.axibase.com/chartlab/7e91b333/4/)
+Name|Example|Description|&nbsp;
```
Comment (rodionos) [inline widgets/text-widget/README.md]: Check all settings with updated 1.08 vscode plugin. Any errors for this widget or other widget?

File: widgets/text-widget/README.md (hunk 1 above)
Comment (kghamilton89) [reply widgets/text-widget/README.md]: Removed

File: widgets/text-widget/README.md (hunk 2 above)
Comment (kghamilton89) [reply widgets/text-widget/README.md]: removed

File: widgets/text-widget/README.md (hunk 3 above)
Comment (kghamilton89) [reply widgets/text-widget/README.md]: All settings check out with the latest plugin version.

**Comment by kghamilton89** [comment]
* All settings run through the latest plugin version (no errors).
* `time-span` replaced with `timespan`.
* Possible values: `false`, `true` added.
* Language standardized.
* Removed setting `label` because it is included in Shared Settings documentation.

**Comment by kghamilton89** [comment]
Table is reformatted. PR is ready.

**Pull request merged**